src/bci_build/package/base-fips/README.md.j2

```diff
@@ -49,7 +49,7 @@ by NIST.
 
 {% else %}
 This base container image is configured with FIPS mode enabled by default, but
-does not include any certified binaries.
+**does not** include any certified binaries.
```

Comment:
I also suggested before that this sentence alone is not of much help.

I would like to see something like:

This base container image is configured with FIPS mode enabled by default, but does not include any certified binaries (e.g., OpenSSL and libgcrypt modules).

With FIPS mode enabled, only FIPS-approved algorithms are allowed. For legal compliance, FIPS mode enabled is not enough without certified binaries.

If you need a certified FIPS image, you should use the LTSS version, which includes OpenSSL and libgcrypt modules that have been interim validated to FIPS 140-3.